specification/communication/data-plane/Microsoft.CommunicationServicesIdentity/2020-01-15-generalAvailability/examples/CreateIdentity.json

```diff
@@ -0,0 +1,14 @@
+{
+  "parameters": {
+    "api-version": "2020-01-15-generalAvailability",
+    "content-type": "application/json",
+    "endpoint": "https://contoso.communication.azure.com/identities"
```

Comment:
nit: `contoso` 👉🏻 `my-resource`
Same for other samples

Reply:
Updated